Dockerfile

```diff
@@ -1,11 +1,20 @@
 # don't change this line
 FROM korney4eg/nginx-php:latest
 
-# your code goes here
-# Make nginx use app.conf configuration to properly render php files
-# Make sure that index.php file is available in browser
-# expose needed ports
+RUN apt update && apt -y upgrade
 
+COPY app.conf /etc/nginx/sites-enabled/app.conf
+RUN chmod +x /etc/nginx/sites-enabled/app.conf
```

Comment:
why do you need execute permissions?

Reply:
As I saw nginx.conf also has execute permission, but now I think all configuration files should be read-only.